lib/content-services/permission-manager/components/add-permission/add-permission.component.html

```diff
@@ -0,0 +1,14 @@
+<adf-add-permission-panel
+    (select)="onSelect($event)">
+</adf-add-permission-panel>
+<div id="add-permission-actions">
```

Comment:
can those id "add-permission-actions and  add-permission-action-button start with adf?